test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/RequestTests.cs

```diff
@@ -76,6 +80,72 @@ public class RequestTests
             }
         }
 
+        [Fact]
+        public async Task LargeMultipartUpload()
```

Comment:
I'll have to disable this on Mono because on Mac it is 32-bit only and this test only really works in 64-bit.